### PR TITLE
Fix k8s example

### PR DESCRIPTION
**Pull request by itspluxstahre**
Fixed the yaml formatting of the k8s example and also added an example for an PVC.

### CODE REVIEW

File: examples/k8s-cronjob.md [hunk 1]
```diff
@@ -20,52 +35,48 @@ kind: CronJob
...
+    
+                # If you don't want to use a PVC you may comment the next two lines, but that will significantly 
+                # affect performance, and is NOT recommended
+                - name: FF_STATE_DIR
+                  value: "/data/"
+                - name: FF_SERVER
+                  valueFrom:
+                    secretKeyRef:
+                      name: fedifetcher
+                      key: server_domain
```
Comment (nanos) [inline examples/k8s-cronjob.md:66]: Why have you removed the `optional: false` setting?

File: examples/k8s-cronjob.md [hunk 2]
```diff
@@ -20,52 +35,48 @@ kind: CronJob
...
+                - name: FF_ACCESS_TOKEN
+                  valueFrom:
+                    secretKeyRef:
                       name: fedifetcher
-                      readOnly: false
```
Comment (nanos) [inline examples/k8s-cronjob.md:65]: why is this no longer read-only?

**Review by nanos — COMMENTED** [review]
Thanks for your PR @itspluxstahre ! Really appreciate you taking the time to contribute to this!

Unfortunately I don't really know k8s, so I'm asking a couple of questions for clarification, to ensure these changes weren't accidental. Can you clarify please?

Thanks!

**Comment by itspluxstahre** [comment]
> Thanks for your PR @itspluxstahre ! Really appreciate you taking the time to contribute to this!
> 
> Unfortunately I don't really know k8s, so I'm asking a couple of questions for clarification, to ensure these changes weren't accidental. Can you clarify please?
> 
> Thanks!

Thanks for the feedback, I will doublecheck these later today and clarify if needed or update the PR. :)

**Comment by nanos** [comment]
Hey @itspluxstahre just wondering if you’ve had a chance to look into this? No rush of course but just wanted to make sure it isn’t forgotten. 

Thanks.

File: examples/k8s-cronjob.md (hunk 1 above)
Comment (itspluxstahre) [reply examples/k8s-cronjob.md:66]: You are correct here, added again.

File: examples/k8s-cronjob.md (hunk 2 above)
Comment (itspluxstahre) [reply examples/k8s-cronjob.md:65]: You are correct here, added again. 

I had two versions of the file, as I did things to the version I use for my cluster that might be confusing in an example as it very specific for the cluster I use.

**Comment by nanos** [comment]
Thanks @itspluxstahre

**Pull request merged**